sphinxcontrib/doxylink/doxylink.py

```diff
@@ -53,6 +57,24 @@ def report_warning(env, msg, docname=None, lineno=None):
     else:
         env.warn(docname, msg, lineno=lineno)
 
+def is_url(str_to_validate):
```

Comment:
Could we add a test for this function?

Reply:
I've added tests just now in 22c226360f7485a4126c6e1fcabe39549ca03e1a.